config/initializers/devise_no_pass.rb

```diff
@@ -0,0 +1,16 @@
+require 'devise/strategies/authenticatable'
+
+module Devise
```

Comment:
This will work only with User model. Not universal, and you hack default strategies. I'm afraid it may have hidden consequences. Should it be another solution based on controller?